main.go

```diff
@@ -4,27 +4,45 @@ import (
 	"github.com/gin-gonic/gin"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/mozilla/doorman/config"
 	"github.com/mozilla/doorman/doorman"
 	"github.com/mozilla/doorman/utilities"
 )
 
+func init() {
+	config.AddLoader(&config.FileLoader{})
+	config.AddLoader(&config.GithubLoader{
+		Token: settings.GithubToken,
+	})
+}
+
 func setupRouter() (*gin.Engine, error) {
 	r := gin.New()
 	// Crash free (turns errors into 5XX).
 	r.Use(gin.Recovery())
 
 	// Setup logging.
+	setupLogging()
+
 	r.Use(HTTPLoggerMiddleware())
 
 	// Setup doorman and load configuration files.
-	w := doorman.NewDefaultLadon(doorman.Config{
-		Sources: config.Sources,
-	})
-	if err := w.LoadPolicies(); err != nil {
+	d := doorman.NewDefaultLadon()
```

Comment:
This can be done after attempting config load. Might as well put all the `doorman` bits together.